homework/01LessonOne.js

```diff
@@ -148,9 +144,8 @@ var notNotTrue = false;
 //
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
-// Your Answer Goes Here:
-//Truthiness is something that evaluates to TRUE. For instance, 5 > 3
-//Falsiness is something that evaluates to FALSE. Fors instance, 5 > 9
+// Your Answer Goes Here: Truthiness value is a value which evaluates to true and a falsiness value is a value which evaluates false. Examples of truthy ---- if (true), if ({}), if ([]), if (12), if ('coding'). Examples of falsy ----- if (false), if (''), if (NaN), if (0), if (null).
+//
 // ****
```

Comment:
Here is some reason for a closer answer -->

https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Very close